qpileup/test/org/qcmg/pileup/MessagesTest.java

```diff
@@ -1,4 +1,4 @@
-package org.qcmg.qpileup;
+package org.qcmg.pileup;
```

Comment:
any reason the package name is changing?

Reply:
as above